core-bundle/src/Crawl/Escargot/Factory.php

```diff
@@ -223,18 +223,14 @@ private function registerSubscribers(Escargot $escargot, array $selectedSubscrib
      */
     private function validateSubscribers(array $selectedSubscribers): array
     {
-        $msg = sprintf(
-            'You have to specify at least one valid subscriber name. Valid subscribers are: %s',
-            implode(', ', $this->getSubscriberNames())
-        );
-
-        if (0 === \count($selectedSubscribers)) {
-            throw new \InvalidArgumentException($msg);
-        }
-
         $selectedSubscribers = array_intersect($this->getSubscriberNames(), $selectedSubscribers);
 
         if (0 === \count($selectedSubscribers)) {
+            $msg = sprintf(
```

Comment:
Why use a one-time variable at all? To bypass the "single line throws" fixer?

```php
throw new \InvalidArgumentException(sprintf('You have to specify at least one valid subscriber name. Valid subscribers are: %s', implode(', ', $this->getSubscriberNames())));
```

Reply:
> To bypass the "single line throws" fixer?

Touché.